doc/docs/modules/ROOT/pages/quickstart.adoc

```diff
@@ -366,3 +374,34 @@ otherwise if `false`:
 ==== Schema
 
 If APOC are installed, schema will be created with `apoc.meta.relTypeProperties`. Otherwise the first 10 (or any number specified by the `schema.flatten.limit` option) results will be flattened and the schema will be create from those properties.
+
+=== Read data by custom Cypher Query
+
+You can specify a Cypher query in this way:
+```scala
+val df = spark.read.format("org.neo4j.spark.DataSource")
+      .option("query", "MATCH (n:Person) WITH n LIMIT 2 RETURN collect(n) AS nodes")
+      .load()
+
+df.show()
+```
+
+The struct of the Dataset returned by the query is influenced by the query itself, in this particular context it could happen
```

Comment:
put a note here -- recommend that individual property fields be returned, rather than returning node, relationship, and path types.  This best maps to spark's type system and yields best results.   So for the cypher query, better than:

```
MATCH (p:Person) RETURN p
```

is

```
MATCH (p:Person)
RETURN id(p) as id, p.name as name
```

Update the quick example to show this kind of usage pattern to encourage people off the bat to do the right thing.

Reply:
Done